securedrop_client/queue.py

```diff
@@ -28,6 +28,38 @@
 logger = logging.getLogger(__name__)
 
 
+class RunnablePriorityQueue(PriorityQueue):
```

Comment:
@cfm Since you've been working on the queues: just leaving a note here that @zekehuntergreen named this class in a consistent manner, and I'm deliberately not poking into the name at this time.

The `Runnable...` class names don't seem very meaningful to me (they seem very jargon and implementation-oriented to start with), but I don't have formed thoughts on this area of the code yet, so I'm not suggesting we change anything at this point. 

Just letting that said so that it's in your radar and for future reference. :slightly_smiling_face: